homeassistant/components/alexa/entities.py

```diff
@@ -404,6 +405,14 @@ def interfaces(self):
         if supported & media_player.SUPPORT_SELECT_SOURCE:
             yield AlexaInputController(self.entity)
 
+        changechannel_features = (
```

Comment:
```suggestion
        change_channel_features = (
```
Or
```suggestion
        channel_features = (
```
For readability.